src/doc/rustdoc/src/unstable-features.md

```diff
@@ -84,6 +84,37 @@ in documentation.
 `#![feature(doc_cfg)]` feature gate. For more information, see [its chapter in the Unstable
 Book][unstable-doc-cfg] and [its tracking issue][issue-doc-cfg].
 
+An extension to this feature called `doc_auto_cfg` was added. With it, you don't need to add
+`#[doc(cfg(..)]` anymore unless you want to override the default behaviour. So if we take the
+previous source code:
+
+```rust
+#![feature(doc_auto_cfg)]
+
+/// Token struct that can only be used on Windows.
+#[cfg(any(windows, doc))]
+pub struct WindowsToken;
+
+/// Token struct that can only be used on Unix.
+#[cfg(any(unix, doc))]
+pub struct UnixToken;
+
+/// Token struct that is only available with the `serde` feature
+#[cfg(feature = "serde")]
+#[derive(serde::Deserialize)]
+pub struct SerdeToken;
+```
+
+It'll render almost the same, the difference being that `doc` will also be displayed. To fix this,
+you can use `doc_cfg_hide`:
```

Comment:
Not a comment about the docs, but shouldn't hiding `doc` be the default behavior?

Reply:
Once https://github.com/rust-lang/rust/pull/92856 is merged, it'll be pretty easy to ignore it as well.